definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState-bailiff-nonprod.json

```diff
@@ -145,5 +145,26 @@
     "CaseStateID": "IssuedToBailiff",
     "UserRole": "caseworker-divorce-courtadmin-la",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "02/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseStateID": "AwaitingBailiffReferral",
+    "UserRole": "[RESPSOLICITOR]",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "02/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseStateID": "AwaitingBailiffService",
+    "UserRole": "[RESPSOLICITOR]",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "02/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseStateID": "IssuedToBailiff",
+    "UserRole": "[RESPSOLICITOR]",
+    "CRUD": "CRU"
```

Comment:
From ticket:

1. RESPSOLICITOR already has CRU permissions to trigger the 'solicitorDraftAOS', ''solicitorUpdateAOS'' and ''solicitorSubmitAOS' events
2. However, RESPSOLICITOR needs to have CRU permissions to the states  'AwaitingBailiffReferral' 'AwaitingBailiffService', 'IssuedToBailiff'